tests/functional/disaster-recovery/regional-dr/test_active_hub_down_and_restore.py

```diff
@@ -0,0 +1,193 @@
+    failover,
+    relocate,
+    restore_backup,
+    create_backup_schedule,
+    get_current_primary_cluster_name,
+    get_current_secondary_cluster_name,
+    get_passive_acm_index,
+    wait_for_all_resources_creation,
+    wait_for_all_resources_deletion,
+    verify_drpolicy_cli,
+    verify_restore_is_completed,
+    get_scheduling_interval,
+)
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.ocs.resources.pod import wait_for_pods_to_be_running
+from ocs_ci.ocs.utils import get_active_acm_index
+from ocs_ci.utility.utils import TimeoutSampler
+
+
+logger = logging.getLogger(__name__)
+
+
+@tier4a
+@turquoise_squad
+class TestActiveHubDownAndRestore:
+    """
+    Test failover and relocate all apps when active hub down and restored RDR
+    """
+
+    def test_hub_recovery_failover_and_relocate(self, nodes_multicluster, dr_workload):
+
+        """
+        Tests to verify failover and relocate all apps when active hub down and restored RDR
+        """
+
+        # acm_obj = AcmAddClusters()
+        # Deploy Subscription and Appset based application
+        rdr_workload = dr_workload(
+            num_of_subscription=1, num_of_appset=1, switch_ctx=get_passive_acm_index()
+        )
+        logger.info(type(rdr_workload))
+        primary_cluster_name = get_current_primary_cluster_name(
+            rdr_workload[0].workload_namespace
+        )
+        secondary_cluster_name = get_current_secondary_cluster_name(
+            rdr_workload[0].workload_namespace
+        )
+        scheduling_interval = get_scheduling_interval(
+            rdr_workload[0].workload_namespace, rdr_workload[0].workload_type
+        )
+        # Create backup-schedule on active hub
+        create_backup_schedule()
+        two_times_scheduling_interval = 2 * scheduling_interval  # Time in minutes
+        wait_time = 300
+        logger.info(f"Wait {wait_time} until backup is taken ")
+        time.sleep(wait_time)
+
+        # Get the active hub nodes
+        logger.info("Getting Active cluster node details")
+        config.switch_ctx(get_active_acm_index())
+        active_hub_index = config.cur_index
+        active_hub_cluster_node_objs = get_node_objs()
+        # ToDo Add verification for dpa and policy
+
+        # Shutdown active hub nodes
+        logger.info("Shutting down all the nodes of active hub")
+        nodes_multicluster[active_hub_index].stop_nodes(active_hub_cluster_node_objs)
+        logger.info(
+            "All nodes of active hub zone are powered off, "
+            f"wait {wait_time} seconds before restoring in passive hub"
+        )
+
+        # Restore new hub
+        restore_backup()
+        logger.info(f"Wait {wait_time} until restores are taken ")
+        time.sleep(wait_time)
+
+        # Verify the restore is completed
+        verify_restore_is_completed()
+
+        # Validate the clusters are imported
+        clusters = [primary_cluster_name, secondary_cluster_name]
```

Comment:
In case of co-situated, primary cluster will not be imported due to it's inaccessibility